axlearn/common/quantized_dot_general/layers.py

```diff
@@ -63,6 +64,14 @@ class ClippingChoice(Enum):
     OUTPUT_ACTIVATION = 1
 
 
+FP8_SCALE_PARAM_NAMES = ["input_scale", "kernel_scale", "output_grad_scale"]
```

Comment:
Consider using enum:

```suggestion
class Fp8ScaleParam(enum.Enum):
    INPUT_SCALE = "input_scale"
    ...
```

This avoids spelling mistakes.

Reply:
Done.